README.md

```diff
@@ -1,175 +1,161 @@
-# Managing HPCS Service Instances using Terraform Modules
+<!-- BEGIN MODULE HOOK -->
 
-This is a collection of modules that make it easier to provision and manage HPCS Instance IBM Cloud Platform:
+<!-- Update the title to match the module name and add a description -->
+# IBM Cloud Hyper Protect Crypto Services
+<!-- UPDATE BADGE: Update the link for the following badge-->
+[![Incubating (Not yet consumable)](https://img.shields.io/badge/status-Incubating%20(Not%20yet%20consumable)-red)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
+[![Build status](https://github.com/terraform-ibm-modules/terraform-ibm-hpcs/actions/workflows/ci.yml/badge.svg)](https://github.com/terraform-ibm-modules/terraform-ibm-hpcs/actions/workflows/ci.yml)
+[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
+[![latest release](https://img.shields.io/github/v/release/terraform-ibm-modules/terraform-ibm-hpcs?logo=GitHub&sort=semver)](https://github.com/terraform-ibm-modules/terraform-ibm-hpcs/releases/latest)
+[![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
-* [Provisioning HPCS Instances](./examples/ibm-hpcs-instance)
-* [Initialising HPCS Instance](./examples/init-using-local)
-* [Managing Keys on HPCS Instance](./examples/ibm-hpcs-kms-key)
+You can use this module to provision and configure an IBM Cloud Hyper Protect Crypto Services instance. There are two ways to use this module:
+* Create Hyper Protect Crypto Services instance and initialize the instance manually.
+* Create and initialize the Hyper Protect Crypto Services instance automatically. It supports only `recovery crypto unit` method of initialization.
 
-## HPCS Initialisation Architecture
+## Create Hyper Protect Crypto Services instance
```

Comment:
I don't understand the difference between `Create Hyper Protect Crypto Services instance` and `Create and initialize the Hyper Protect Crypto Service instance`

Reply:
@SirSpidey 
`Create Hyper Protect Crypto Services instance` -> it creates the service instance only and then user has to initialize manually.
`Create and initialize the Hyper Protect Crypto Service instance` -> this includes the service instance creation and initialization automatically through module.